src/lang/de-de.json

```diff
@@ -12,6 +12,100 @@
+      "privacy_policy": "<a href='https://www.plentymarkets.com/de/datenschutz/' target='_blank' class='text-primary text-appearance'>https://www.plentymarkets.com/de/datenschutz/</a>",
+      "lifespan": "Session"
+    },
+
+    "cookies_session": {
+      "provider": "plentymarkets",
+      "purpose": "Der Session-Cookie behält die Zustände des Benutzers bei allen Seitenanfragen bei.",
+      "privacy_policy": "<a href='https://www.plentymarkets.com/de/datenschutz/' target='_blank' class='text-primary text-appearance'>https://www.plentymarkets.com/de/datenschutz/</a>",
+      "lifespan": "Session"
+    },
+
+    "cookies_csrf": {
+      "provider": "plentymarkets",
+      "purpose": "Der CSRF-Cookie dient dazu, Cross-Site Request Forgery-Angriffe zu verhindern.",
+      "privacy_policy": "<a href='https://www.plentymarkets.com/de/datenschutz/' target='_blank' class='text-primary text-appearance'>https://www.plentymarkets.com/de/datenschutz/</a>",
+      "lifespan": "Session"
+    }
+  },
+
+  "cookies_statistics-label": "Statistik",
+  "cookies_statistics": {
+    "amount": "1",
+    "description": "Statistik-Cookies erfassen Informationen anonym. Diese Informationen helfen uns zu verstehen, wie unsere Besucher unsere Website nutzen.",
+    "cookies_google_analytics_label": "Google Analytics",
+
+    "cookies_google_analytics": {
+      "provider": "Google Ireland Limited, Irland oder Google LLC, USA",
+      "purpose": "Zur Verbesserung unserer Produkte und Werbemaßnahmen",
+      "privacy_policy": "<a href='https://policies.google.com/privacy' target='_blank' class='text-primary text-appearance'>https://policies.google.com/privacy</a>",
+      "lifespan": "2 Jahre"
+    }
+  },
+
+  "cookies_external-media-label": "Externe Medien",
+  "cookies_external_media": {
+    "amount": "1",
+    "description": "Inhalte von Videoplattformen und Social Media Plattformen werden standardmäßig blockiert. Wenn Cookies von externen Medien akzeptiert werden, bedarf der Zugriff auf diese Inhalte keiner manuellen Zustimmung mehr.",
+    "cookies_youtube_label": "Youtube",
+
+    "cookies_youtube": {
+      "provider": "Google Ireland Limited, Irland oder Google LLC, USA",
+      "purpose": "Zur Darstellung wichtiger Inhalte auf der Webseite",
+      "privacy_policy": "<a href='https://policies.google.com/privacy' target='_blank' class='text-primary text-appearance'>https://policies.google.com/privacy</a>",
+      "lifespan": "2 Jahre"
+    }
+  },
+
+  "cookies_functional-label": "Funktional",
+  "cookies_functional": {
+    "amount": "1",
+    "description": "Diese Cookies ermöglichen, dass die von Nutzern getroffenen Auswahlmöglichkeiten und bevorzugte Einstellungen (z.B. das Deaktivieren der Sprachweiterleitung) gespeichert werden können.",
+    "cookies_preferred_language_label": "Automatische Spracherkennung",
+    
+    "cookies_preferred_language": {
```

Comment:
Where is this used for example?

Reply:
Should be in src/partials/cookie-further-settings.hbs, starting at line 260.